src/plugins/intel_gna/memory/gna_memory.hpp

```diff
@@ -131,7 +132,7 @@ class GNAMemory : public GNAMemRequestsQueue {
                 // std::cout << "  [binded=" << rTypeToStr(re._type) << ", ptr=" << re._ptr_out <<"]\n";
                 visitor(reference, re);
                 // primitive loop check
-                if (re._ptr_in == re._ptr_out) continue;
+                if (re.IsInPtrEqualToOutPtrOf(re)) continue;
```

Comment:
this helper makes the code more complicated, we are comparing _ptr_in and _ptr_out in one memory request and in one place only.

Reply:
will remove this helper